internal/response/exists.go

```diff
@@ -5,14 +5,14 @@ import (
 )
 
 type exists struct {
-	count int
+	count uint32
 }
 
 func Exists() *exists {
 	return &exists{}
 }
 
-func (r *exists) WithCount(n int) *exists {
+func (r *exists) WithCount(n uint32) *exists {
```

Comment:
Technically this is also of type `imap.Seq` -- an exists response returns the sequence number of the last message in the mailbox

Reply:
Fixed.